controllers/storagecluster/storageclient.go

```diff
@@ -126,3 +142,19 @@ func (s *storageClient) updateClientConfigMap(r *StorageClusterReconciler, names
 
 	return nil
 }
+
+// getCephNetworkAnnotationValue returns the network annotation value for the given StorageCluster NetworkSpec.
+func getCephNetworkAnnotationValue(cephNetworkSpec *rookCephv1.NetworkSpec, scNamespace string) (string, error) {
+	if len(cephNetworkSpec.Selectors) != 0 {
+		networkSelectionElement, err := cephNetworkSpec.GetNetworkSelection(scNamespace, rookCephv1.CephNetworkType(cephPublicNetworkType))
+		if err != nil || networkSelectionElement == nil {
```

Comment:
looking at the `GetNetworkSelection`, `nil` seems to be a valid value for `networkSelectionElement` as well, in that scenario if we requeue via error are we expecting user to update the value?

Reply:
yes, if multus is enabled then network selections are must to be present